bitsandbytes/backends/cuda.py

```diff
@@ -414,11 +439,13 @@ def quantize_4bit(
         A: torch.Tensor,
         absmax: Optional[torch.Tensor] = None,
         out: Optional[torch.Tensor] = None,
-        blocksize=64,
+        blocksize: Optional[int] = None,
         compress_statistics=False,
         quant_type: Literal["fp4", "nf4"] = "fp4",
         quant_storage=torch.uint8,
     ) -> Tuple[torch.Tensor, QuantState]:
+        if blocksize is None:
+            blocksize = 64 if not HIP_ENVIRONMENT else 128
```

Comment:
Is there a short explanation we can add here to explain why this is the default, and likewise below why 64 is not supported?

Reply:
Its because of warpsize difference between AMD and NVIDIA GPUs. I have added comments - https://github.com/TimDettmers/bitsandbytes/pull/1207/commits/410f4998907cd7b4b7978b4a19c98afea66a80da